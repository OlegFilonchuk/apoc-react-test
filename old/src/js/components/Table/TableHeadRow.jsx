import React from 'react';
import PropTypes from 'prop-types';

import { markAsTableHeadRow } from './Table';

export default function TableHeadRow({ children, className }) {
  return (
    <thead className={className}>
      <tr>{children}</tr>
    </thead>
  );
}

markAsTableHeadRow(TableHeadRow);

TableHeadRow.propTypes = {
  children: PropTypes.node.isRequired,
  className: PropTypes.string
};

TableHeadRow.defaultProps = {
  className: ''
};
