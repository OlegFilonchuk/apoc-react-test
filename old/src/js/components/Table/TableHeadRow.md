Purpose of this component is to be used together with [TableHeadCell](#tableheadcell). See example there.
