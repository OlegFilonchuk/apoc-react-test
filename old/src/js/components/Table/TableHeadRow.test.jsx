import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import TableHeadRow from './TableHeadRow';

describe('<TableRow />', () => {
  it('should match snapshot', () => {
    const row = shallow(
      <TableHeadRow>
        <td>Content</td>
        <td>Content</td>
      </TableHeadRow>
    );

    expect(toJson(row)).toMatchSnapshot();
  });
});
