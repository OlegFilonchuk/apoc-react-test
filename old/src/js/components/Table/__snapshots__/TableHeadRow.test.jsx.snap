// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableRow /> should match snapshot 1`] = `
<thead
  className=""
>
  <tr>
    <td>
      Content
    </td>
    <td>
      Content
    </td>
  </tr>
</thead>
`;
